### PR TITLE
Add docker stack and complement existing

### PR DESCRIPTION
**Pull request by joao10lima**
As described in the issue #23, using as example the GitHub starters CI, add to the Pipelinit the following stacks:
- Docker stack
  - Lint stage
  - Build stage
- Java stack
  - Build stage with Gradle 
- Complement python stack
  - Django and PyTest
  - Black as formatter and flake8 adjusted
- Complement JavaScript stack
  - Add a stage test

To test:
- Check if the unit tests pass 
- Try to generate the Pipelinit CI accordingly to each stack using the sample projects https://github.com/pipelinit/pipelinit-cli/tree/feat/github-starter-stacks#examples

image
image
image
image

### CODE REVIEW

File: core/plugins/stack/python/django.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { IntrospectFn } from "../../../types.ts";
+
+export const introspect: IntrospectFn<boolean> = async (context) => {
+  // Search for project manage.py to define as a Django project
+  for await (const _file of context.files.each("**/manage.py")) {
+    return true;
+  }
+  return false;
```
Comment (oesgalha) [inline core/plugins/stack/python/django.ts:8]: Use the `includes` helper to do this: 
```suggestion
  return await context.files.includes("**/manage.py");
```

File: core/templates/github/javascript/test.yaml [hunk 2]
```diff
@@ -14,4 +14,22 @@ jobs:
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '<%= it.runtime.version %>'
+          cache: '<%= it.packageManager.name %>'
+      - run: <%= it.packageManager.name %> ci
```
Comment (oesgalha) [inline core/templates/github/javascript/test.yaml:33]: This should be `npm ci` for npm or `yarn` for yarn.

File: core/templates/github/python/lint.yaml [hunk 3]
```diff
@@ -9,5 +12,7 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: <%= it.version %>
-      - run: python -m pip install flake8
-      - run: flake8
+
+      - run: python -m pip install pip flake8 black
+      - run: black . --check
+      - run: flake8 --ignore E203,E501,W503 .
```
Comment (oesgalha) [inline core/templates/github/python/lint.yaml:18]: Can you add a comment here explaining why those should be ignored?

**Review by oesgalha — CHANGES_REQUESTED** [review]
Thank you!

I've made some suggestions below, can you take a look?

---

In the README file, Increase the rowspan from the cells with "Coming soon" from 8 to 13. This will make the last columns render with all the rows.

**Comment by joao10lima** [comment]
> Thank you!
> 
> I've made some suggestions below, can you take a look?
> 
> In the README file, Increase the rowspan from the cells with "Coming soon" from 8 to 13. This will make the last columns render with all the rows.

Thanks for the comments, I've changed accordingly and updated the README example section text

**Review by oesgalha — APPROVED** [review]

**Pull request merged**